charts/hlf-k8s/Chart.yaml

```
@@ -15,7 +15,7 @@
 apiVersion: v1
 name: hlf-k8s
 home: https://substra.org/
-version: 1.3.2
+version: 1.4
```

Actually it's 1.4.0, because 1.4 is not a string in Yaml terms, sorry about that
```suggestion
version: 1.4.0
```

---

Redone :)